src/go/k8s/pkg/resources/node_port_service.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package resources contains reconciliation logic for redpanda.vectorized.io CRD
+package resources
```

Comment:
why do we have this in all files? 🤔

Reply:
Sorry I don't understand the question. 

The strait answer is the golang requires that every go file has declared package at the top of the file.

Only guess that you would like to know why there is a comment above the package. The linter complains with the following comments:
> pkg/resources/configmap.go:10:1: should have a package comment, unless it's in another file for this package (golint)

Reply:
yeah I meant the comment, sorry. Uhm, that's really ugly... to have this duplicated line in every file

Reply:
There are 2 option:

1. Remove the linter
2. Change the package godoc

I don't know what you would like to see in the comment.

Reply:
oh yeah, it's not blocker for this PR. 

```
should have a package comment, unless it's in another file for this package (golint)
```

that sounds like one should be enough. Anyway, yeah. Let's solve this somewhere else. I just keep seeing this in all PRs recently